From patchwork Tue Sep  6 19:48:39 2022
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Liam Howlett <Liam.Howlett@Oracle.com>
X-Patchwork-Id: 12968053
Return-Path: <owner-linux-mm@kvack.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from kanga.kvack.org (kanga.kvack.org [205.233.56.17])
	by smtp.lore.kernel.org (Postfix) with ESMTP id D6666ECAAA1
	for <linux-mm@archiver.kernel.org>; Tue,  6 Sep 2022 19:48:45 +0000 (UTC)
Received: by kanga.kvack.org (Postfix)
	id 45B4A6B0073; Tue,  6 Sep 2022 15:48:45 -0400 (EDT)
Received: by kanga.kvack.org (Postfix, from userid 40)
	id 3BB7D6B0074; Tue,  6 Sep 2022 15:48:45 -0400 (EDT)
X-Delivered-To: int-list-linux-mm@kvack.org
Received: by kanga.kvack.org (Postfix, from userid 63042)
	id 0FD918D0001; Tue,  6 Sep 2022 15:48:45 -0400 (EDT)
X-Delivered-To: linux-mm@kvack.org
Received: from relay.hostedemail.com (smtprelay0014.hostedemail.com
 [216.40.44.14])
	by kanga.kvack.org (Postfix) with ESMTP id F3A0E6B0073
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 15:48:44 -0400 (EDT)
Received: from smtpin04.hostedemail.com (a10.router.float.18 [10.200.18.1])
	by unirelay08.hostedemail.com (Postfix) with ESMTP id D13A514083C
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 19:48:44 +0000 (UTC)
X-FDA: 79882698168.04.B8ABAAF
Received: from mx0a-00069f02.pphosted.com (mx0a-00069f02.pphosted.com
 [205.220.165.32])
	by imf14.hostedemail.com (Postfix) with ESMTP id 61B9D100064
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 19:48:44 +0000 (UTC)
Received: from pps.filterd (m0246627.ppops.net [127.0.0.1])
	by mx0b-00069f02.pphosted.com (8.17.1.5/8.17.1.5) with ESMTP id
 286Id9gU019965;
	Tue, 6 Sep 2022 19:48:43 GMT
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=oracle.com;
 h=from : to : cc :
 subject : date : message-id : references : in-reply-to : content-type :
 content-transfer-encoding : mime-version; s=corp-2022-7-12;
 bh=DUu2GQ4uP+RysYobIs//6ncaOCZvvl9onwhVgCsrCgQ=;
 b=xlSFnxzx1pMphdg/iPYUpFtjRiSQnufnaqFF92dzrAZr5eEhx5nfWR+uRbbL+uIXiXUr
 bzB93kcMHEj7+SpTJcMMzAAPHvbE8qnzpN9dRmqn8oa3mnIRDaMoQiahinfWkgWujtaB
 14+nFFdqudxxx84/13Z0cwrZrweArzjmgBTtF2AXbFcXwe7yXlONioJdxcQgbU+8fhCd
 Ih2IlpTQWaNnXKc+VWhpHT/9qzEB1UiRGEcxbNFZqigjY8bPx91Xu19eB/rpqbkj9NJx
 /6RwccT4uH/flVmmIZKxtgzSZgF33wrfh/CdEnhilMzxV5Z1upyFON0RHO8mwYVh3Wun IA==
Received: from phxpaimrmta02.imrmtpd1.prodappphxaev1.oraclevcn.com
 (phxpaimrmta02.appoci.oracle.com [147.154.114.232])
	by mx0b-00069f02.pphosted.com (PPS) with ESMTPS id 3jbwh1exj7-1
	(version=TLSv1.2 cipher=ECDHE-RSA-AES256-GCM-SHA384 bits=256 verify=OK);
	Tue, 06 Sep 2022 19:48:43 +0000
Received: from pps.filterd
 (phxpaimrmta02.imrmtpd1.prodappphxaev1.oraclevcn.com [127.0.0.1])
	by phxpaimrmta02.imrmtpd1.prodappphxaev1.oraclevcn.com (8.17.1.5/8.17.1.5)
 with ESMTP id 286HHdKI037489;
	Tue, 6 Sep 2022 19:48:42 GMT
Received: from nam12-bn8-obe.outbound.protection.outlook.com
 (mail-bn8nam12lp2171.outbound.protection.outlook.com [104.47.55.171])
	by phxpaimrmta02.imrmtpd1.prodappphxaev1.oraclevcn.com (PPS) with ESMTPS id
 3jbwc32vns-1
	(version=TLSv1.2 cipher=ECDHE-RSA-AES256-GCM-SHA384 bits=256 verify=OK);
	Tue, 06 Sep 2022 19:48:42 +0000
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=mURKGuL//GbVhscrCbjd6iRGs5st5h3isG37a/I9jorThG0ZLo+vYDhezVpZ5QIcbQqsoRq8B5FZNv/L/SYJQZqX0piWtLElxdLR5OUvjA9urS4P2QHJPQDcFBUY9nSpxaBiIdnnRK8GcAtUB4GkIk6rK14JCycAEWGsUZreD00Nx9hKB19xolBzqvpOHUMXuWvixixtIJntfwWRmF67enf6dCYQBvAVoyOT8rZjBWBw39HJVGPjpQgQ6i4blLHDcOgFBdHEgl+3z7IPrcrYCo7EESbyufB/dL5tEDRycAMg3zG+ynIGXROI7eaiGNQVIDM8+geUv+ajFYBRjv/FAg==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-AntiSpam-MessageData-ChunkCount:X-MS-Exchange-AntiSpam-MessageData-0:X-MS-Exchange-AntiSpam-MessageData-1;
 bh=DUu2GQ4uP+RysYobIs//6ncaOCZvvl9onwhVgCsrCgQ=;
 b=Yfa9k+1ux4FtRdCuongiaDKmyBi28NN7Wr9FevWGN11zQC7UiJiHbQt3TX5cPHZvlZj7Sq902RTiBbZCRnsPAmbsxVAvrVuxqpc1I+GJ5kE5QZTQEU5l0vMAVKUw6QxYLEYXCL1yjjB9W4ynDC2B3nu/EUDj1MJHeO8pHVVoK9aWCILonaN0f121yuk7wx7l+Qjwger6gybYon9g71xI/GEvE7QeI0NUOJXcR2/cCQUmAH8Y+6+BGsmFZbV6Lf8a55j5YjOvLht9A5f7T/dlHupqKRGYGPfNVodSyMw7mC5OR8IctXj0txntO85bPT8lbxZ9r/RLdFPEJOs6OHiKbQ==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=oracle.com; dmarc=pass action=none header.from=oracle.com;
 dkim=pass header.d=oracle.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=oracle.onmicrosoft.com; s=selector2-oracle-onmicrosoft-com;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=DUu2GQ4uP+RysYobIs//6ncaOCZvvl9onwhVgCsrCgQ=;
 b=bsB1jqb1Nmcz0ISIKIFLcPzZguVTq7QPBiNKjQPW29UQpg8r1GVj4udTitCrx8qNiwV/b5mHkhms76DVISD09zYZEVNjBiZ0sMzHA8rj/w+vgA61fXx1jlAggQvMu2yl9iHkney9KHw46nMPZ8lb47Xo41ceDcI2mm9x1qz+sRc=
Received: from SN6PR10MB3022.namprd10.prod.outlook.com (2603:10b6:805:d8::25)
 by DM4PR10MB6183.namprd10.prod.outlook.com (2603:10b6:8:8b::22) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.5588.10; Tue, 6 Sep
 2022 19:48:40 +0000
Received: from SN6PR10MB3022.namprd10.prod.outlook.com
 ([fe80::a420:3107:436d:d223]) by SN6PR10MB3022.namprd10.prod.outlook.com
 ([fe80::a420:3107:436d:d223%5]) with mapi id 15.20.5588.018; Tue, 6 Sep 2022
 19:48:40 +0000
From: Liam Howlett <liam.howlett@oracle.com>
To: "maple-tree@lists.infradead.org" <maple-tree@lists.infradead.org>,
        "linux-mm@kvack.org" <linux-mm@kvack.org>,
        "linux-kernel@vger.kernel.org"
	<linux-kernel@vger.kernel.org>,
        Andrew Morton <akpm@linux-foundation.org>
CC: Liam Howlett <liam.howlett@oracle.com>
Subject: [PATCH v14 02/70] radix tree test suite: add pr_err define
Thread-Topic: [PATCH v14 02/70] radix tree test suite: add pr_err define
Thread-Index: AQHYwimprsTHENaGm0+u+A2ABlRcmQ==
Date: Tue, 6 Sep 2022 19:48:39 +0000
Message-ID: <20220906194824.2110408-3-Liam.Howlett@oracle.com>
References: <20220906194824.2110408-1-Liam.Howlett@oracle.com>
In-Reply-To: <20220906194824.2110408-1-Liam.Howlett@oracle.com>
Accept-Language: en-US
Content-Language: en-US
X-MS-Has-Attach: 
X-MS-TNEF-Correlator: 
x-mailer: git-send-email 2.35.1
x-ms-publictraffictype: Email
x-ms-office365-filtering-correlation-id: 700363c2-a9e9-4bff-6dcc-08da9040cbff
x-ms-traffictypediagnostic: DM4PR10MB6183:EE_
x-ms-exchange-senderadcheck: 1
x-ms-exchange-antispam-relay: 0
x-microsoft-antispam: BCL:0;
x-microsoft-antispam-message-info: 
 eg5RvKiaS1IX5RFeCYpK3s1aP9D69XrNefXScIlozHMVkeDFD0o5HUsaZihSKsZRn+Ha2+Jg72A/3AjLCFmZieQYKdLX0M47mxISBDsiCzh01kM2ZFziDkZayJqxAjleWgbUo9etBltdtz3LwvUwPnLRrv8bHOX8lYmVi6nSp8H8TJObZXvUtpWZbxQ7X5qzGcSEPV/jksSUT+85BFtpUQTKwhYEE1oj5+dtoundCrPdkL19Yj/mcP2+9Q1lmMdY19p6Xzh6+uUjaUmyPDrMRabDkFFHeQqqMWYF1592YY6R/6qLLSixcIKR/lW81XlkIuhkaNnjHfs9vHWVbue4hhaK1b6jIsZW4i3jThNtWwOAvuqDfzI5nsVCwY2v1uIUm6DR2PM/1ZTJYm1SODrMIV9P+TwH4ykof1hiNYtem9eRqeXknKA8bnjCw6wbUZAKgqxlV9cTsJXJLzW3RKhRudo6nuYyvYwZ/ndCoKEFLevlKJzLDcUSHK4Rv79zlEcKXUs7psdp0xbyNOELXTNbN4JgorymL7g6pQ/9OwW7a/ic2y1fOeTbZHrYQoEyDep2u9I+FkbnLRIGEiRgSOKWJgXrKSSrkM0k+wI57ho3SPQoHizQqWJ622sXViRSLuobF+o3tga9qpAGZRbAfMVJ9nGGPzhioSWthnijmSEDw3TY8ihNbxEqw4/Ilq4Nl1/O0l+3azopfeZTAzzA9rbPjPkCK5FVaXQ9rCk83Lxcf/KlvrseZVreXXIh0OJrB4jZ
x-forefront-antispam-report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR10MB3022.namprd10.prod.outlook.com;PTR:;CAT:NONE;SFS:(13230016)(39860400002)(366004)(376002)(396003)(346002)(136003)(66556008)(38100700002)(122000001)(8936002)(5660300002)(76116006)(66946007)(66446008)(66476007)(2906002)(4744005)(8676002)(91956017)(36756003)(2616005)(186003)(107886003)(86362001)(1076003)(110136005)(44832011)(71200400001)(38070700005)(26005)(64756008)(41300700001)(6506007)(6512007)(6486002)(316002)(478600001)(4326008);DIR:OUT;SFP:1101;
x-ms-exchange-antispam-messagedata-chunkcount: 1
x-ms-exchange-antispam-messagedata-0: =?iso-8859-1?q?CwzvmU7jrX3wnCSL5kyKEMv?=
	=?iso-8859-1?q?xGIrIp3511zcg4oTddGxoeVCEQKWxCUEo2BhdhYP9ARqdNv48lkT4sEUN+rC?=
	=?iso-8859-1?q?Lwtw3nM29ENxKbM7+ylEpLZyb2kMo0ipb/qk9xPpNl9nNMHnYOZsYAmgDUX/?=
	=?iso-8859-1?q?oH/PH+1CWqvTKjnoY2+qUG6Qyi/n1KCOiO7TVPNdG/pbrpaNbU5JDBinxpFt?=
	=?iso-8859-1?q?65K1GWHCzX+gcXiDUMQkVSYwyGg+jDismuIfl0oXTW1sT4skshbf7lPKYj/c?=
	=?iso-8859-1?q?aX7uvXHR84IU6lSdDZ9NKS9Oo5S2N2OfaM2fsY/2sELB/2FwAG4E0UwAwhp7?=
	=?iso-8859-1?q?CjtLy+xRtgugV4dOnH/kp2c5aDE8JdcKd6Efq2mG2YWavix5U7RLEN8xaa+q?=
	=?iso-8859-1?q?rfRb4nqP/stvBJ0Mnkht49syL6jL7DS4ZI4H5a8b1KZ8gRAIddt8RfGclMmq?=
	=?iso-8859-1?q?NU9SNZ3y7z9ogGHrh0pqjmecdybAkvHl2vN/gLS7Sa6sg3TSDszlbpRVlkIb?=
	=?iso-8859-1?q?L5cV43i183YMHwvuGQpenLBRg/8uvPJTBgeO4dmK5+JUnN6riiaFemDW4wH5?=
	=?iso-8859-1?q?PdPHCwnTxYCeVQSjwJH9rmP6ku13CGgkPESKo0ggBDrLQnfAYYmD0DD2v/iw?=
	=?iso-8859-1?q?vjNR4p26Nsn3F+WE4Q7KmvBCAxpAT/l0Sro5dYUXHY+CY64RcZWGUFavbnDT?=
	=?iso-8859-1?q?ndrDsEBXxOL0F5rYVQ7UFKyBpKJGagLjPDgJyNNTeM4+UEDjl6T1ji2SPim8?=
	=?iso-8859-1?q?ItrhBRaALNmXzIJtJwl8L2bIegF3aBU5qdjCyh75Bf2UBMTujj7+vgDvUlrj?=
	=?iso-8859-1?q?TD9epPne40OuqvPID4lc0nKi4POjml0Dz6kXtgviEsOJ94Fc3OYfiH8O0zY8?=
	=?iso-8859-1?q?kGY1TaguVT8vzJtd360Tj/aNMVKkIC5A/1Ws0nnu8QhC6wn4nP0GnlbYIszX?=
	=?iso-8859-1?q?P1fgo2/2gayTozy5BqZr/Mm9oAwh8FR35NfUzZCT7qfo3q7XFkgWsSNQXvf0?=
	=?iso-8859-1?q?7HtWxS/89VRC/vZoZHT3MlaAqYFXrNmuZ6RWzBohKuI1/vCnI85UG0e12G5x?=
	=?iso-8859-1?q?D5fjEx5ZwCdfMrR0/Oc5quc8ot5+eKc1UU1mGfFdG5cyY/HcSMDAQcl+Hq16?=
	=?iso-8859-1?q?djOOgrEOBTZQANygSh2vS/dn6qni4vGJk0CtbY2tFQaS8lztO+NIJjcqMQAp?=
	=?iso-8859-1?q?YrnYN96/+Edo7sf3e7Pkzg+NMElNM8JDKNp+5GkMikLsqdLa//76ncbvk3Eh?=
	=?iso-8859-1?q?QhQaAPsuB6x0cw8+el1VLvY0Q87JeuWUGjZwRNZY0uIoYCe63vt89XuvEUxs?=
	=?iso-8859-1?q?+eiLR9CkaMA5zIb5y21lhElClpPx5znWpwIMlkDCeQOS6kK/UzaQ4GZwPLui?=
	=?iso-8859-1?q?eDScy4ddwEpR5vTVAeQh/2D+j2gswCAZG/BiDsrqVaiisOy6Oy6oysxsAYWz?=
	=?iso-8859-1?q?9yN8KmzpBkOXvZXvMJSeC9bvi1RsIszR66x2PLRrRr+umv5cOxF2AICXDPE/?=
	=?iso-8859-1?q?oYAazXze9e0oKRTuglkH+wU/mHsLsfjhTrVKoSh4MOL8j9MHqGPXQjEBhlW8?=
	=?iso-8859-1?q?1omHXvxInuk4udaaeG/X+i9UjspqozC380mn43a8a48+8sf+4CuvHrAurtcF?=
	=?iso-8859-1?q?xco/rXuhlxTIIJSiK0w1NiTU5RWYoL1jdargTtQ=3D=3D?=
MIME-Version: 1.0
X-OriginatorOrg: oracle.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-AuthSource: SN6PR10MB3022.namprd10.prod.outlook.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 700363c2-a9e9-4bff-6dcc-08da9040cbff
X-MS-Exchange-CrossTenant-originalarrivaltime: 06 Sep 2022 19:48:39.8824
 (UTC)
X-MS-Exchange-CrossTenant-fromentityheader: Hosted
X-MS-Exchange-CrossTenant-id: 4e2c6054-71cb-48f1-bd6c-3a9705aca71b
X-MS-Exchange-CrossTenant-mailboxtype: HOSTED
X-MS-Exchange-CrossTenant-userprincipalname: 
 9CwkjmxkZI4kU36yR/cMlRRTihVBr+LP893TQk92XdL319ySAdskbs+PukpafSkV+Ib6R2xqXx6+N4QzpdBvoA==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: DM4PR10MB6183
X-Proofpoint-Virus-Version: vendor=baseguard
 engine=ICAP:2.0.205,Aquarius:18.0.895,Hydra:6.0.528,FMLib:17.11.122.1
 definitions=2022-09-06_09,2022-09-06_02,2022-06-22_01
X-Proofpoint-Spam-Details: rule=notspam policy=default score=0 bulkscore=0
 suspectscore=0 phishscore=0
 mlxlogscore=999 spamscore=0 adultscore=0 mlxscore=0 malwarescore=0
 classifier=spam adjust=0 reason=mlx scancount=1 engine=8.12.0-2207270000
 definitions=main-2209060091
X-Proofpoint-GUID: k1ehD-0_gMW-apE3HWdscaailQlRrDJH
X-Proofpoint-ORIG-GUID: k1ehD-0_gMW-apE3HWdscaailQlRrDJH
ARC-Message-Signature: i=2; a=rsa-sha256; c=relaxed/relaxed;
 d=hostedemail.com;
	s=arc-20220608; t=1662493724;
	h=from:from:sender:reply-to:subject:subject:date:date:
	 message-id:message-id:to:to:cc:cc:mime-version:mime-version:
	 content-type:content-type:
	 content-transfer-encoding:content-transfer-encoding:
	 in-reply-to:in-reply-to:references:references:dkim-signature;
	bh=DUu2GQ4uP+RysYobIs//6ncaOCZvvl9onwhVgCsrCgQ=;
	b=KH1CD/5OC8nbfS2ipUPUFxRObt4iB+DlKi1k9koacXkWZ0hWvfU2FBOmUirAewPkNRUQWn
	6zW8eVd4PR2wiAPQ/p8sPBu3uhestMPIDUikozGCUqyu80rmuonaH5UbzfdSvNdpv3M2l1
	sr0BFaIQiM2keIW6p6Rp/tLpi0aUKF4=
ARC-Authentication-Results: i=2;
	imf14.hostedemail.com;
	dkim=pass header.d=oracle.com header.s=corp-2022-7-12 header.b=xlSFnxzx;
	dkim=pass header.d=oracle.onmicrosoft.com
 header.s=selector2-oracle-onmicrosoft-com header.b=bsB1jqb1;
	spf=pass (imf14.hostedemail.com: domain of liam.howlett@oracle.com designates
 205.220.165.32 as permitted sender) smtp.mailfrom=liam.howlett@oracle.com;
	arc=pass ("microsoft.com:s=arcselector9901:i=1");
	dmarc=pass (policy=none) header.from=oracle.com
ARC-Seal: i=2; s=arc-20220608; d=hostedemail.com; t=1662493724; a=rsa-sha256;
	cv=pass;
	b=uxT0AHYkj40eaXhzy0gKbrlXGHcQUlvGmOAJvkLYmnIDJJ3qjeVpOeErzGK3G0a+/HJOYX
	a5PNPoPcZHtRRGpdHLs3Hl2yYM6RRuWfVRiZrmtM5GwnO2rP2qZXdWvCQ0YN1KtxxWmG48
	b72K0Pz9fKL415OBQyPv9hMNagfBTUQ=
X-Stat-Signature: pwa8grtfokr9swj48umjisjojsqa7soc
X-Rspam-User: 
X-Rspamd-Queue-Id: 61B9D100064
Authentication-Results: imf14.hostedemail.com;
	dkim=pass header.d=oracle.com header.s=corp-2022-7-12 header.b=xlSFnxzx;
	dkim=pass header.d=oracle.onmicrosoft.com
 header.s=selector2-oracle-onmicrosoft-com header.b=bsB1jqb1;
	spf=pass (imf14.hostedemail.com: domain of liam.howlett@oracle.com designates
 205.220.165.32 as permitted sender) smtp.mailfrom=liam.howlett@oracle.com;
	arc=pass ("microsoft.com:s=arcselector9901:i=1");
	dmarc=pass (policy=none) header.from=oracle.com
X-Rspamd-Server: rspam01
X-HE-Tag: 1662493724-502949
X-Bogosity: Ham, tests=bogofilter, spamicity=0.000000, version=1.2.4
Sender: owner-linux-mm@kvack.org
Precedence: bulk
X-Loop: owner-majordomo@kvack.org
List-ID: <linux-mm.kvack.org>

From: "Liam R. Howlett" <Liam.Howlett@Oracle.com>

define pr_err to printk

Signed-off-by: Liam R. Howlett <Liam.Howlett@Oracle.com>
---
 tools/testing/radix-tree/linux/kernel.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/radix-tree/linux/kernel.h b/tools/testing/radix-tree/linux/kernel.h
index 39867fd80c8f..c5c9d05f29da 100644
--- a/tools/testing/radix-tree/linux/kernel.h
+++ b/tools/testing/radix-tree/linux/kernel.h
@@ -14,6 +14,7 @@
 #include "../../../include/linux/kconfig.h"
 
 #define printk printf
+#define pr_err printk
 #define pr_info printk
 #define pr_debug printk
 #define pr_cont printk

